packages/neuron-ui/src/utils/hooks/index.ts

```diff
@@ -437,7 +437,7 @@ export const useGlobalNotifications = (
 ) => {
   useEffect(() => {
     const lastVersion = syncRebuildNotification.load()
-    if (isReadyByVersion(+CONSTANTS.SYNC_REBUILD_SINCE_VERSION, lastVersion ? +lastVersion : null)) {
+    if (lastVersion !== null && isReadyByVersion(CONSTANTS.SYNC_REBUILD_SINCE_VERSION, lastVersion)) {
```

Comment:
`lastVersion = null` is handled in `isReadyByVersion` so `lastVersion !== null` could be removed.

Reply:
I change the logic by https://github.com/nervosnetwork/neuron/pull/2517/commits/13e76aa05896e42ef95d9b638dbcd06eabb9b6bd#diff-82dfd1ad092d59d026219b46ac797b96978458927dd355ec48258ea36472b437
Because I think if users download the latest neuron, we shouldn't show the migrate dialog. And save the last version when the dialog show whenever.